### DISCUSSION TITLE
Screenshot upload regression, high resolution issue

### DISCUSSION THREAD

**Discussion by oliviertassinari** [community/community#74738] [Repositories]
### Select Topic Area

Bug

### Body

I have found the same issue as https://github.com/orgs/community/discussions/27715 but for screenshots.

I'm not sure if it's a recent regression of GitHub UI or if it's related to the migration to macOS v14.0.0 but uploading screenshot to GitHub comments is super frustrating now. I have to constantly manually fix the markdown to add a width attribute.

See how this screenshot is too large.

![Screenshot 2023-11-08 at 22 47 56](https://github.com/community/community/assets/3165635/ad6ec805-996a-4083-ba5b-7f3792be5149)

it should render at the correct scale 1:1

<img src="https://github.com/community/community/assets/3165635/ad6ec805-996a-4083-ba5b-7f3792be5149" width="188" />

**Comment by gitstua** [comment]
Thanks for raising @oliviertassinari - I have added labels to this.

Can you clarify where you see the issue above please?

Is this on a `MarkDown file in the code view`, in an `issue` or in a `discussion`?

Did the [workaround](https://github.com/orgs/community/discussions/27715#discussioncomment-6443628) work for you posted in the other thread?

**Reply by oliviertassinari** [reply]
I can reproduce on GitHub discussions, GitHub issues, and GitHub PRs.

> Did the [workaround](https://github.com/orgs/community/discussions/27715#discussioncomment-6443628) work for you posted in the other thread?

This can be an approximation but it usually doesn't create reliable "look and feel", the scale is usually not 1:1, but more 1:1.1354

**Comment by oliviertassinari** [comment]
Looking at logs, I could debug the problem. It's related to https://apple.stackexchange.com/questions/466762/mac-screenshot-resolution-on-macbook-pro-changed-from-144-to-72-dpi-after-upgrad. The "X Resolution" field in the EXIF of the screenshot I take is wrong.

I can work around the problem by no longer using the native screenshot tool of macOS and instead by using [Shottr](https://shottr.cc/) that GitHub continues to read exif data correctly from.

**Comment by maphew** [comment]
I have this same issue with screenshots originating on Win10 and Win11 that have hi-dpi displays. The png do not appear to have EXIF data similar to the above. Example:

![note-icon-change](https://github.com/user-attachments/assets/7a46ea87-8894-4964-ac1e-cd66543b2788)

I see this with screenshots taken with Greenshot and the native Snipping Tool.
